libpod/boltdb_state.go

```diff
@@ -47,6 +55,9 @@ func NewBoltState(path, lockDir string, runtime *Runtime) (State, error) {
 		if _, err := tx.CreateBucketIfNotExists(nameRegistryBkt); err != nil {
 			return errors.Wrapf(err, "error creating name-registry bucket")
 		}
+		if _, err := tx.CreateBucketIfNotExists(nsRegistryBkt); err != nil {
```

Comment:
(At some point it would be nice to have a documentation of all the bucket data formats _somewhere_.)

Reply:
Added a header comment explaining DB layout